cmd/create_prs/create_prs.go

```diff
@@ -86,11 +86,6 @@ func run(c *cobra.Command, _ []string) {
 	skippedCount := 0
 	errorCount := 0
 	for _, repo := range dir.Repos {
-		if sleep > 0 {
```

Comment:
@leszekbulawa instead of moving the sleep stance altogether (and risking situations where exceptions bypass it), could you introduce a step count and skip the initial sleep if we're at the first step?

```go
for i, repo := range dir.Repos {
  if i > 0 and sleep > 0 {
```

Reply:
Good idea, done.